packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -118,21 +125,51 @@ function WorklistDataGrid(props) {
     rowHeaderIndex,
   } = props;
 
+  // Default column size constraints
+  const defaultColumnMinimumWidth = 60;
+  const defaultColumnMaximumWidth = 300;
+
+  // Initialize column width properties
+  const initializeColumn = (column) => {
+    const newColumn = { ...column };
+    newColumn.width = column.width || defaultColumnWidth;
+    newColumn.minimumWidth = column.minimumWidth || defaultColumnMinimumWidth;
+    newColumn.maximumWidth = column.maximumWidth || defaultColumnMaximumWidth;
+
+    return newColumn;
+  };
+
+  const displayedColumns = (hasSelectableRows ? [WorklistDataGridUtils.ROW_SELECTION_COLUMN] : []).concat(columns);
+  const [dataGridColumns, setDataGridColumns] = useState(displayedColumns.map((column) => initializeColumn(column)));
+
+  // Manage column resize
+  const [tableHeight, setTableHeight] = useState(0);
+  const [activeIndex, setActiveIndex] = useState(null);
+  const activeColumnPageX = useRef(0);
+  const activeColumnWidth = useRef(200);
+  const tableWidth = useRef(0);
+
   const grid = useRef();
   const focusedRow = useRef(0);
   const focusedCol = useRef(0);
   const ariaLiveMsg = useRef();
 
   const [currentSelectedCell, setCurrentSelectedCell] = useState(null);
   const [, setRowSelectionModeMessage] = useState();
```

Comment:
Not related to your changes, but what is the purpose of this state variable?

Reply:
This state variable will cause the rerender to occur so that the aria live region will be updated.  We might change the implementation slightly in the future.